### PR TITLE
RTRT - Email inline footer - AB test to 50%

### PR DESCRIPTION
**Pull request by gtrufitt**
Open up the inline footer to 50% of logged-out UK audience.

There are a few prerequisites to sign-off before merging this:
- [x] ~~Update tracking with campaign codes to ensure we're able to test engagement~~ ( me )(Seperate campaign codes are not required initially with one segment, but this will need to be passed through the API to enable us to track across A/B tests)
- [x] Are we tracking all we need? We have hits for 'subscribe' and 'success shown' ( @amyhughes )
- [x] Any other design updates? ( @superfrank - outside of footer adjustments which can roll out soon after if not ready before)
- [x] Update actual email to link to Register page ( @crifmulholland @desbo )
- [x] Adding to the correct email list ( @crifmulholland )

(Feel free to add any other requirements before merging this)

### CODE REVIEW

**Comment by amyhughes** [comment]
Are we tracking in test/not in test?

On 27 November 2015 at 12:03, Gareth Trufitt <notifications@github.com>
wrote:

> Open up the inline footer to 50% of logged-out UK audience.
>
> There are a few prerequisites to sign-off before merging this:
>
>    - Update tracking with campaign codes to ensure we're able to test
>    engagement ( me )
>    - Are we tracking all we need? We have hits for 'subscribe' and
>    'success shown' ( @amyhughes <https://github.com/amyhughes> )
>    - Any other design updates? ( @superfrank
>    <https://github.com/superfrank> - outside of footer adjustments which
>    can roll out soon after if not ready before)
>    - Update actual email to link to Register page ( @crifmulholland
>    <https://github.com/crifmulholland> @desbo <https://github.com/desbo> )
>    - Adding to the correct email list ( @crifmulholland
>    <https://github.com/crifmulholland> )
>
> (Feel free to add any other requirements before merging this)
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/guardian/frontend/pull/11218
> Commit Summary
>
>    - Remove AB test update methods
>    - Update AB test metadata, canRun restrictions and variant ID
>    - Revert "Remove AB test update methods"
>    - Remove modification
>    - Update title
>
> File Changes
>
>    - *M* applications/app/views/emailFragment.scala.html
>    <https://github.com/guardian/frontend/pull/11218/files#diff-0> (2)
>    - *M*
>    static/src/javascripts/projects/common/modules/experiments/tests/rtrt-email-form-inline-footer.js
>    <https://github.com/guardian/frontend/pull/11218/files#diff-1> (28)
>
> Patch Links:
>
>    - https://github.com/guardian/frontend/pull/11218.patch
>    - https://github.com/guardian/frontend/pull/11218.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/11218>.
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396

File: static/src/javascripts/projects/common/modules/experiments/tests/rtrt-email-form-inline-footer.js [hunk 1]
```diff
@@ -56,29 +58,29 @@ define([
         };
 
         this.id = 'RtrtEmailFormInlineFooter';
```
Comment (johnduffell) [inline static/src/javascripts/projects/common/modules/experiments/tests/rtrt-email-form-inline-footer.js]: just be sure that keeping the test name the same won't mess things up - if it's already put it in people's localstorage you might end up with unbalanced sample sizes

Comment (gtrufitt) [inline static/src/javascripts/projects/common/modules/experiments/tests/rtrt-email-form-inline-footer.js]: That is a very good point, everyone who has visited recently will end up with notintest in their localstorage. I'll change the test name, thanks!

Comment (gtrufitt) [inline static/src/javascripts/projects/common/modules/experiments/tests/rtrt-email-form-inline-footer.js]: Updated the test!

**Comment by desbo** [comment]
:+1:

**Pull request merged**